features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/operationComponents/OperationGovernance.jsx

```diff
@@ -152,58 +152,60 @@ rate limiting per operation
             <Grid item md={6} />
             <Grid item md={1} />
             <Grid item md={5}>
-                <TextField
-                    id='operation_scope'
-                    select
-                    disabled={disableUpdate}
-                    fullWidth
-                    label='Operation scope'
-                    value={getOperationScopes(operation, spec)[0]}
-                    onChange={({ target: { value } }) => operationsDispatcher({
-                        action: 'scopes',
-                        data: { target, verb, value: [value] },
-                    })}
-                    helperText='Select a scope to control permissions to this operation'
-                    margin='dense'
-                    variant='outlined'
-                >
-                    <MenuItem
-                        value=''
-                        dense
+                { operation['x-auth-type'] && operation['x-auth-type'].toLowerCase() !== 'none' ? (
+                    <TextField
+                        id='operation_scope'
+                        select
+                        disabled={disableUpdate}
+                        fullWidth
+                        label={api.scopes.length !== 0 ? 'Operation scope' : 'No scope available'}
```

Comment:
Add localization for all rendering texts. Follow the wiki https://github.com/wso2/carbon-apimgt/wiki/How-internationalization-(i18n)-works-in-API-Manager-React-Apps